datalab/datalab_session/data_operations/median.py

```diff
@@ -1,5 +1,11 @@
+from io import BytesIO
 from datalab.datalab_session.data_operations.data_operation import BaseDataOperation
-
+from datalab.datalab_session.util import store_fits_output, find_fits
+import numpy as np
+from astropy.io import fits
+import logging
+log = logging.getLogger()
```

Comment:
We're gonna want two blank lines between the imports and other code according to pep8 https://peps.python.org/pep-0008/#blank-lines